### PR TITLE
added two email verification pages with components

### PR DESCRIPTION
**Pull request by shagariboy**
I worked on two pages:

- verify email/sign-up page
- after verification/go to login page

### CODE REVIEW

File: components/EmailVerified/EmailVerified.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import styles from "styles/Verify-email.module.scss";
+
+class EmailVerified extends Component {
```
Comment (copy-ninja1) [inline components/EmailVerified/EmailVerified.tsx:4]: let's stick to hooks instead

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]

File: styles/Verify-email.module.scss [hunk 2]
```diff
@@ -0,0 +1,368 @@
+@import url('http://fonts.cdnfonts.com/css/inter');
+
```
[the remark this answers is not in the document]
Comment (shagariboy) [reply styles/Verify-email.module.scss:2]: updated to Tailwind css

File: components/EmailVerified/EmailVerified.tsx (hunk 1 above)
Comment (shagariboy) [reply components/EmailVerified/EmailVerified.tsx]: Made updates.

**Comment by shagariboy** [comment]
I would like to remove the "styles/Verify-email.module.scss" file. It has a lot of bugs and is not useful anymore.

**Pull request merged**